GFYPL

```diff
@@ -6,6 +6,7 @@ GO FUCK YOURSELF PRIVATE LICENCE v1.0
 4. IF YOU SEE THIS LICENSE, GO FUCK YOURSELF.
 5. YOU CANNOT ASK THIS TO BE RELICENCED, GO FUCK YOURSELF.
 6. YOU CANNOT DOWNLOAD THIS CODE. GO FUCK YOURSELF.
-7. THERE IS NO WARRANTY. GO FUCK YOURSELF.
-8. I'M NOT LIABLE FOR ANYTHING. GO FUCK YOURSELF.
-9. IF YOU SEE THIS LICENSE, GO FUCK YOURSELF.
+7. YOU CANNOT REVERSE ENGINEER THE PRODUCT. GO FUCK YOURSELF.
```

Comment:
```suggestion
7. YOU CANNOT REVERSE ENGINEER THIS CODE. GO FUCK YOURSELF.
```

Reply:
Thank you for your holy suggestion, licence lord